include/zenoh-pico/transport/raweth/config.h

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2022 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+#ifndef ZENOH_PICO_RAWETH_CONFIG_H
+#define ZENOH_PICO_RAWETH_CONFIG_H
+
+#include <stdbool.h>
+
+#include "zenoh-pico/protocol/core.h"
+#include "zenoh-pico/system/link/raweth.h"
+#include "zenoh-pico/transport/transport.h"
+#include "zenoh-pico/utils/result.h"
+
+#if Z_FEATURE_RAWETH_TRANSPORT == 1
+
+typedef struct {
+    _z_keyexpr_t _keyexpr;
+    uint16_t _vlan;  // vlan tag (pcp + dei + id)
```

Comment:
Previously you defined `_zp_vlan_tag_t` type. Should we use the same type here? Or have `_zp_vlan_tag_t` simply a `uint16_t`?